### DISCUSSION TITLE
Why Menu `trigger="hover"` is not accessible?

### DISCUSSION THREAD

**Discussion by italodeandra** [mantinedev/mantine#2389] [Feedback]
> Menu with trigger="hover" is not accessible – users that navigate with keyboard will not be able to use it
> [Source](https://mantine.dev/core/menu/#show-menu-on-hover)

This seems like it could be easily fixable by also accepting the click event. Was this by design or is it a limitation? Why?

**Comment by rtivital** [comment]
I'm not really sure, maybe we should add it

**Reply by italodeandra** [reply]
Okay. I created an issue for it https://github.com/mantinedev/mantine/issues/2390

**Comment by mg-aceik** [comment]
Hi @rtivital I'm just taking a look into this one.
On the issue you said:

> I've checked this issue – menu cannot be opened with click for hover trigger because focus will not be trapped. It does not make much sense to open the menu in this case, as it will not be possible to control it with keyboard.

So I think the use case here is as follows:

* Keyboard only user loads the page. Uses standard keyboard controls to select the menu target and presses enter which traps focus in the dropdown and they can navigate the focusable items within the dropdown until they escape out.
* Mouse user A loads the page. On hover on the menu target, the dropdown opens and they can use the mouse to select menu items. (No need to trap focus, I think).
* Mouse user B loads the page. Hovers on the menu target which opens the dropdown. They wish the dropdown to stay open though and trap focus so they click on the menu target which traps focus.

You can see this behaviour [here](https://www.powercor.com.au/) as an example (although it does automatically open the dropdown on focus which I am not necessarily suggesting we do here). If you agree I can take a stab at trying to implement it.

**Reply by rtivital** [reply]
Sure, you are welcome to submit a PR
